packages/cli/src/commands/validatorgroup/member.ts

```diff
@@ -40,8 +45,20 @@ export default class ValidatorGroupRegister extends BaseCommand {
 
     if (res.flags.accept) {
       await displaySendTx('addMember', validators.addMember((res.args as any).validatorAddress))
-    } else {
-      await displaySendTx('addMember', validators.removeMember((res.args as any).validatorAddress))
+    } else if (res.flags.remove) {
+      await displaySendTx(
+        'removeMember',
+        validators.removeMember((res.args as any).validatorAddress)
+      )
+    } else if (res.flags.reorder != null) {
+      await displaySendTx(
+        'reorderMember',
+        await validators.reorderMember(
+          res.flags.from,
+          (res.args as any).validatorAddress,
```

Comment:
Just curious, what is the default type of `res.args`?

Reply:
```ts
{
    [name: string]: any;
}
```
don't know why